HAP transform: Update README.md and add sample notebook

ian-cho opened the pull request:
update readme for hap transform

## Related issue number (if any).
https://github.com/IBM/data-prep-kit/issues/753

ian-cho committed:
Update README.md

update readme for hap transform

shahrokhDaijavad commented:
Thank you, @ian-cho. The README looks good. Three requests:
1) Can you please add a "Contributors" section with your name and email at the top like this: https://github.com/IBM/data-prep-kit/tree/update-doc_quality-readme/transforms/language/doc_quality/python? 
2) The link to the test_hap.py file seems to be broken. 
3) We have now finalized the format of the example notebook. Can you please create one like this: https://github.com/IBM/data-prep-kit/blob/update-doc_quality-readme/transforms/language/doc_quality/doc_quality.ipynb? Of course, the content will be similar to what you have in hap_local_python.py and some text from your README about the parameters. Thank you.

shahrokhDaijavad reviewed (changes requested):
Please see my comments.

ian-cho commented:
@shahrokhDaijavad the links seem to be broken:

- https://github.com/IBM/data-prep-kit/tree/update-doc_quality-readme/transforms/language/doc_quality/python?
- https://github.com/IBM/data-prep-kit/blob/update-doc_quality-readme/transforms/language/doc_quality/doc_quality.ipynb

shahrokhDaijavad commented:
Sorry, @ian-cho. After I gave you the links to the branch above, the branch was merged, so the new links are in the dev branch at:
https://github.com/IBM/data-prep-kit/blob/dev/transforms/language/doc_quality/python/README.md
and 
https://github.com/IBM/data-prep-kit/blob/dev/transforms/language/doc_quality/doc_quality.ipynb

ian-cho committed:
Add files via upload

ian-cho committed:
Update README.md

1. added contributor
2. added hap notebook

ian-cho commented:
@shahrokhDaijavad Hi, I updated the PR as requested. Please review, thanks.
1. added contributor 
2. For the link, replace it with notebook
3. added the hap python notebook.

shahrokhDaijavad commented on transforms/universal/hap/python/README.md:
```diff
@@ -48,6 +56,20 @@ python hap_local_python.py
 You will obtain the output file `test1.parquet` in the output directory.
 
+### Code example
+[notebook](../hap_python.ipynb)
```
The link to the notebook is broken. The correct link is: ./hap_python.ipynb, because the notebook is in the python directory.

shahrokhDaijavad reviewed (commented):
Thank you, @ian-cho, for developing the minimal notebook that is functionally correct. For the consistency of your notebook with others, before the current cell one that imports the necessary packages, can you please add what we add in other notebooks, similar to what is here: https://github.com/IBM/data-prep-kit/blob/dev/transforms/language/doc_quality/doc_quality.ipynb?

ian-cho committed:
Add files via upload

ian-cho commented:
@shahrokhDaijavad Thank you for further suggestions. Before code cells, I added markdown content and configurations for the hap transforms just as _doc_quality.ipynb_ did. Please review.

shahrokhDaijavad commented:
Thank you, @ian-cho. The notebook looks good. And what about the broken link in the README above?

ian-cho committed:
Update README.md

ian-cho commented:
> Thank you, @ian-cho. The notebook looks good. And what about the broken link in the README above?

@shahrokhDaijavad Sorry, I missed that. Fixed it

shahrokhDaijavad reviewed (approved):
LGTM. Thank you, @ian-cho.

touma-I committed:
fix layout for commands in first cell

Signed-off-by: Maroun Touma <touma@us.ibm.com>

touma-I committed:
Merge branch 'dev' into ian-cho-patch-1
